content/3_Silver/Two_Pointers.mdx

```diff
@@ -280,3 +280,103 @@ public class Books {
 ## Problems
 
 <Problems problems="ad" />
+
+## Quiz
+
+<Quiz>
+  <Quiz.Question>
+    What is the two pointers algorithm?
+    <Quiz.Answer>
+      Keeping track of two indices to find a subarray that satisfies some condition.
+      <Quiz.Explanation>
+        Incorrect. The indices don't necessarily have to represent a subarray, they could just represent elements in the array.
+	      
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      Keeping track of two indices to find a pair of indices that satisfy some condition.
+      <Quiz.Explanation>
+        Correct.
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      A way to answer range sum queries in $\mathcal{O}(1)$ time.
+      <Quiz.Explanation>
+        Incorrect. You might be thinking of [Prefix Sums](/silver/prefix-sums), another silver topic.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      A divide and conquer algorithm that splits an array into two halves to efficiently find an element satisfying a certain condition.
+      <Quiz.Explanation>
+        Incorrect.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    What should go in place of `???`?
+
+	```
+	/*
+	 * find two elements at distinct positions in an array 
+	 * which add up to a target sum
+	 */
+	  
+	nums.sort()
+
+	l = 0
+	r = n - 1
+
+	while (???) {
+		if (nums[l] + nums[r] == target) {
+			found pair
+			break
+		}
+		else if (nums[l] + nums[r] > target) {
+			r--
+		} else {
+			l++
+		}
+	}
+	```
+    <Quiz.Answer>
+      `nums[r] - nums[l] != 0`
```

Comment:
add support for mulitiple languages?

Reply:
ill do cpp and python, but i don't know java :)

Reply:
the code blocks won't work for python though, `||` is `or` in python. should i leave it, or revert to latex

Reply:
ill resolve for now, since i think it's pretty self-explanatory, but lmk if i should change it